execution_engine_testing/tests/src/test/explorer/faucet.rs

```diff
@@ -347,7 +348,7 @@ fn should_fund_existing_account() {
 
     assert_eq!(
         user_purse_balance_after,
-        user_purse_balance_before + one_distribution - user_account_initial_balance
+        user_purse_balance_before + one_distribution - user_account_initial_balance + refund
```

Comment:
Looks like we're testing the refund as a side effect in various different tests. Do you think there's a merit in adding a dedicated test like `fn should_refund_proper_amount()` or similar?

Reply:
that's true, we could maybe set the refund to zero percent for these tests and then add a separate dedicated test that uses a non-zero refund, wdyt @rafal-ch?

Reply:
I'd be ok with just an additional test dedicated to testing the refund. I think we don't need to revert the other changes you made.